### PR TITLE
feature(extension): Add issue ignore (hide / show) functionality to checker

### PR DESCRIPTION
**Pull request by drjoho**
* Extension UI Feature

### This PR is related to the following issue(s): 
- 
#936
#1664
#1666 
#1667 
#1685

### Additional information can be found here: 
- 

### Testing reference: 
- 
See Design Document
https://www.figma.com/file/intddqthciXilHQCLWppQ7/a11y-checker?type=design&node-id=428-78341&mode=design&t=8pT6zTqUoUWAUIDh-0

### I have conducted the following for this PR: 
- [x] I validated this code in Chrome and FF 
- [x] I validated this fix in my local env
- [x] I provided details for testing
- [ ] This PR has been reviewed and is ready for test  
- [ ] I understand that the title of this PR will be used for the next release notes.

### CODE REVIEW

**Review by tombrunet — DISMISSED** [review]

**Comment by tombrunet** [comment]
@ErickRenteria Please confirm that it all looks good in dark mode also.

**Review by tombrunet — APPROVED** [review]

**Comment by ErickRenteria** [comment]
Still needs some work

File: accessibility-checker-extension/src/ts/devtools/components/reports/reportRules.tsx [hunk 1]
```diff
@@ -81,6 +82,7 @@ export class ReportRules extends React.Component<ReportProps> {
                 group.children.sort((a, b) => UtilIssue.valueToOrder(a.value)-UtilIssue.valueToOrder(b.value));
             }
         }
+        console.log("rowData = ",rowData);
```
Comment (tombrunet) [inline accessibility-checker-extension/src/ts/devtools/components/reports/reportRules.tsx:85]:
```suggestion
```

File: accessibility-checker-extension/src/ts/devtools/components/reportTreeGrid.tsx [hunk 2]
```diff
@@ -845,6 +861,7 @@ export class ReportTreeGrid<RowType extends IRowGroup> extends React.Component<R
                                 ReportTreeGrid.bgcontroller.setIgnore(url, this.state.checkedIssues, ignoreAction !== "Show");
                                 this.setState({checkedIssues: []});
                                 this.props.onFilterToolbar(true);
+                                this.props.onResetFilters();
```
Comment (tombrunet) [inline accessibility-checker-extension/src/ts/devtools/components/reportTreeGrid.tsx:864]: I don't think we want to do this

**Review by tombrunet — CHANGES_REQUESTED** [review]

**Review by tombrunet — DISMISSED** [review]

**Comment by drjoho** [comment]
Ready for review. Show / hide regular, Keyboard mode, and Summary.

**Review by tombrunet — DISMISSED** [review]

**Comment by drjoho** [comment]
@ErickRenteria you can do the review all I did was comment out some content that Design did not want.

**Comment by ErickRenteria** [comment]
Tested on Chrome, FF, and Edge browser in the macOS. No major issue found
Tested on Chrome and FF in the windows env and it looks good. Feature ready to go.
Validated

**Review by ErickRenteria — APPROVED** [review]

**Pull request merged**